### ISSUE TITLE
Device portal in Windows Terminal

### ISSUE THREAD

**Issue by Qantex** [microsoft/terminal#5049]
# Description of the new feature/enhancement
**Windows 10 users currently have the ability to use the Device Portal to control a device on the same connected network (WLAN). This function could also be programmed in the Windows terminal to solve problems faster.**

# Proposed technical implementation details (optional)

**Comment by zadjii-msft** [comment]
I don't think this is a bad idea, but I think this makes more sense as an extension rather than something built directly in to the Terminal.

I'll tag this issue up to reflect that :)

**Comment by Qantex** [comment]
Thanks you very much! ❤😄
